deps/build.jl

```diff
@@ -1,6 +1,7 @@
 using Compat
 
-include(joinpath(@__DIR__, "..", "src", "util", "logging.jl"))
+using Logging
+parse(Bool, get(ENV, "DEBUG", "false")) && global_logger(SimpleLogger(global_logger().stream, Logging.Debug))
```

Comment:
Right, there's not a system for configuration yet, but what you're doing here is a very common pattern for this kind of thing which needs some kind of support. Actually I'm not sure I'll even have time for doing something nice for configuration in 1.0 - there's still a lot of other arguably-breaking-but-probably-not-feature-frozen things which need straightening out in Base, while configuration can be done in a package.

Reply:
Sure, I can work with an env flag and global logger for now.